src/odemis/driver/andorcam3.py

```diff
@@ -916,6 +916,12 @@ def updateTemperatureVA(self):
             else:
                 status_str = ""
             logging.debug(u"Temp is %.2f°C%s", temp, status_str)
+
+            # Check if the status is "Fault" and set the state accordingly
+            if status == "Fault":
+                self.state._set_value(HwError("Camera Temperature fault detected"), force_write=True)
```

Comment:
I'd slightly change the message, as logically I think it's more the *cooling* which has a fault than the *temperature*. So something like "Camera cooling fault detected".